mozilla-release/toolkit/locales/en-US/chrome/mozapps/downloads/downloads.properties

```diff
@@ -32,14 +32,14 @@ offlineCancelDownloadsAlertTitle=Cancel All Downloads?
 offlineCancelDownloadsAlertMsg=If you go offline now, 1 download will be canceled. Are you sure you want to go offline?
```

Comment:
I checked. It should be correct: 
offlineCancelDownloadsAlertMsg=Wenn du jetzt offline gehst, wird 1 Download abgebrochen. Bist du sicher, dass du offline gehen möchtest?